greeting/android/build.gradle

```diff
@@ -1,17 +1,42 @@
-apply plugin: 'kotlin-platform-jvm'
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-platform-android'
 
 // Specify a group and a version of the library to access it in Android Studio.
 // By default the project directory name is used as an artifact name thus the full dependency
 // description will be 'org.greeting:android:1.0'
 group = 'org.greeting'
 version = 1.0
 
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        minSdkVersion 15
+        targetSdkVersion 27
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
 dependencies {
+    expectedBy project(':greeting:common')
+	
     // Specify Kotlin/JVM stdlib dependency.
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    // Specify dependency on a common project for Kotlin multiplatform build.
-    expectedBy project(':greeting:common')
+    implementation 'com.android.support:appcompat-v7:27.1.1'
```

Comment:
looks like we don't need this dependency in sample

Reply:
Done